datadog_checks_dev/datadog_checks/dev/tooling/commands/manifest.py

```diff
@@ -14,6 +14,38 @@
 from ...compat import JSONDecodeError
 from ...utils import basepath, file_exists, read_file, write_file
 
+ATTRIBUTES = {
+    "mandatory": [
```

Comment:
Let's make each of these a set constant then have:

`ALL_ATTRIBUTES = REQUIRED_ATTRIBUTES | OPTIONAL_ATTRIBUTES`